src/main/java/com/klarna/hiverunner/sql/cli/CommandShellEmulator.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2015 Klarna AB
```

Comment:
We should update this to 2015-2018 if it contains code you've moved around from earlier, otherwise just set it to 2018.